### PR TITLE
Added method for getting public key ids from signature

### PR DESCRIPTION
**Pull request by danhab99**
Continuing from #1326 , I kinda broke it because I committed to master...

> Resolving issue #1325 by adding a method for getting the KeyIDs of all the issuers of a given signature

### CODE REVIEW

File: src/signature.js [hunk 1]
```diff
@@ -51,6 +51,19 @@ export class Signature {
...
     return armor(enums.armor.signature, this.write(), undefined, undefined, undefined, config);
   }
+
+  /**
+   * Returns an array of KeyIDs of all of the issuers who created this signature
+   * @returns {Array<KeyID>} The Key IDs of the signing keys
+   */
+  getSigningKeyIDs() {
+    const keyIDs = [];
+    const signatureList = this.packets.filterByTag(enums.packet.signature)
```
Comment (twiss) [inline src/signature.js:61]:
```suggestion
    const signatureList = this.packets.filterByTag(enums.packet.signature);
```

File: test/general/signature.js [hunk 2]
```diff
@@ -650,6 +650,31 @@ hUhMKMuiM3pRwdIyDOItkUWQmjEEw7/XmhgInkXsCw==
 -----END PGP SIGNATURE-----
 `;
 
+
+  it("Retrieve the issuer ID of a signature", async function () {
```
Comment (twiss) [inline test/general/signature.js:654]:
```suggestion
  it("Retrieve the issuer Key ID of a signature", async function () {
```

File: test/general/signature.js [hunk 3]
```diff
@@ -668,6 +693,7 @@ kCNcH9WI6idSzFjuYegECf+ZA1xOCjS9oLTGbSeT7jNfC8dH5+E92qlBLq4Ctt7k
     await expect(openpgp.readSignature({ armoredSignature })).to.be.rejectedWith(/Missing signature creation time/);
   });
 
+
```
Comment (twiss) [inline test/general/signature.js:696]:
```suggestion
```

**Review by twiss — COMMENTED** [review]
Thanks! Some last nitpicks:

File: openpgp.d.ts [hunk 4]
```diff
@@ -22,6 +22,8 @@ export function generateSessionKey(options: { encryptionKeys: PublicKey[], date?
 export function decryptKey(options: { privateKey: PrivateKey; passphrase?: string | string[]; config?: PartialConfig }): Promise<PrivateKey>;
 export function encryptKey(options: { privateKey: PrivateKey; passphrase?: string | string[]; config?: PartialConfig }): Promise<PrivateKey>;
 export function reformatKey(options: { privateKey: PrivateKey; userIDs?: UserID|UserID[]; passphrase?: string; keyExpirationTime?: number; config?: PartialConfig }): Promise<KeyPair>;
+export function revokeKey(options: { key: PrivateKey, reasonForRevocation?: { flag?: enums.reasonForRevocation, string?: String }, config?: PartialConfig }): Promise<PrivateKey>;
+export function revokeKey(options: { key: PublicKey, revocationCertificate?: String, config?: PartialConfig }): Promise<PublicKey>;
```
Comment (twiss) [inline openpgp.d.ts:26]: These will come in #1324, could you leave them out of this PR for now?

File: test/general/signature.js [hunk 5]
```diff
@@ -650,29 +650,29 @@ hUhMKMuiM3pRwdIyDOItkUWQmjEEw7/XmhgInkXsCw==
...
 `;
 
-  it("Retrieve the issuer Key ID of a signature", async function () {
-    const { privateKeyArmored, publicKeyArmored } = await openpgp. generateKey({
-        type: "ecc", // Type of the key, defaults to ECC
-        curve: "curve25519", // ECC curve name, defaults to curve25519
-        userIDs: [{ name: "name", email: "test@email.com"}], // you can pass multiple user IDs
-        passphrase: "password", // protects the private key
-      });
+  it("Retrieve the issuer ID of a signature", async function () {
```
Comment (twiss) [inline test/general/signature.js:653]: They are really called Key IDs :) See https://datatracker.ietf.org/doc/html/rfc4880#section-3.3.
```suggestion
  it("Retrieve the issuer Key ID of a signature", async function () {
```

File: openpgp.d.ts (hunk 4 above)
Comment (danhab99) [reply openpgp.d.ts]: Yup, that's because I merged from my master

**Review by twiss — APPROVED** [review]

**Pull request merged**